clang/test/CodeGenSYCL/offload-fp32-div-spirvbackend.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_cc1 -fsycl-is-device -emit-llvm -triple spirv64-unknown-unknown \
```

Comment:
Yeah, I think you just need to generate IR that crashes codegen now, put it to llvm/test directory and run llc on it. Please don't create a *clang* test that doesn't expose a problem in *clang* .